src/SafeDNS/SettingsClient.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace UKFast\SDK\SafeDNS;
+
+use UKFast\SDK\SafeDNS\Entities\Settings;
+
+class SettingsClient extends Client
+{
+    const SETTINGS_MAP = [
+        'custom_soa_allowed' => 'customSoaAllowed',
+        'custom_base_ns_allowed' => 'customBaseNsAllowed',
+        'custom_axfr' => 'customAxfr',
+        'delegationAllowed'
+    ];
+
+    public function getSettings()
```

Comment:
To match the naming format in the rest of the SDK, should this function just be called `get`? We already have the scope of the `Settings` class so that is slightly redundant

```suggestion
    public function get()
```